src/arroz-chaufa.md

```diff
@@ -18,7 +18,7 @@ Peruvian-chinese dish. Easy to cook just add and mix everything.
 - Soy Sauce
 - Welsh Onion
 
-![Ingredients](pix/arroz_chaufa-1.webp)
+![Ingredients](pix/arroz-chaufa-1.webp)
```

Comment:
```suggestion
![Ingredients](data/pix/arroz-chaufa-1.webp)
```
Fixed Error

Reply:
Same as above. `data` should not be there.